frame/nfts/src/functions.rs

```diff
@@ -36,12 +36,20 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 	) -> DispatchResult {
 		let collection_details =
 			Collection::<T, I>::get(&collection).ok_or(Error::<T, I>::UnknownCollection)?;
-		ensure!(!collection_details.is_frozen, Error::<T, I>::Frozen);
 		ensure!(!T::Locker::is_locked(collection, item), Error::<T, I>::Locked);
```

Comment:
Would be good to split the `Locked` error into `CollectionLocked` and `ItemLocked` so that UIs can give better feedback to the user.

Reply:
Done